src/pooling.jl

```diff
@@ -154,9 +164,13 @@ end
 
 
 """
-    meanpool(x, k::NTuple; pad=0, stride=k)
+    meanpool(x, k::NTuple{N, Integer}; pad=0, stride=k)
 
 Perform mean pool operation with window size `k` on input tensor `x`.
+
+* `x` and `k`: Usually, ndim(x) ∈ [3, 5], length(k) ∈ [1, 3], s.t. ndim(x) == length(k) + 2
+* `pad`: See [`pad_zeros`](@ref) for details.
+* `stride`: Stride for each spatial axis. `k` as default if not present.
```

Comment:
Same here:
```suggestion
* `stride`: Either an integer, or a tuple the same length as `k`. Default is `k`.
```

Reply:
Sorry, my bad. I missed this before. And now I add this following this review.